content/en/examples/application/job/cronjob.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   jobTemplate:
     spec:
       template:
+        metadata:
+          labels:
+            lovely_label: "something_wonderful"
```

Comment:
My concern: does the person reading this page understand what resource this label will apply to?
- the CronJob
- every Job that the cronjob controller makes on behalf of this CronJob
- the Pods that the job controller makes on behalf of those Jobs


If the labelling isn't obvious either self-evidently or through a comment, I worry that it might actually confuse more readers than it helps. Worse, some of those people won't realize their misunderstanding for some time.